.github/workflows/test.yaml

```diff
@@ -0,0 +1,45 @@
+name: TF Test
```

Comment:
@gberenice can you please add this to the module template so that these tests run on each new "Initial PR" of a new module?